accounts/service_test.go

```diff
@@ -100,23 +100,69 @@ func (m *mockLnd) assertInvoiceRequest(t *testing.T, addIndex,
 	}
 }
 
-func (m *mockLnd) assertNoPaymentRequest(t *testing.T) {
+// SubscribeInvoices allows a client to subscribe to updates of newly
+// added/settled invoices.
+func (m *mockLnd) SubscribeInvoices(_ context.Context,
+	req lndclient.InvoiceSubscriptionRequest) (<-chan *lndclient.Invoice,
+	<-chan error, error) {
+
+	if m.invoiceSubscriptionErr != nil {
+		return nil, nil, m.invoiceSubscriptionErr
+	}
+
+	m.invoiceReq <- req
+
+	return m.invoiceChan, m.invoiceErrChan, nil
+}
+
+type mockRouter struct {
```

Comment:
I think we should add a bit more to this commit message. I'm not certain what's happening here. I think you're removing functionality from the LND mock and putting it into a new mock called `mockRouter`.

Is that split related to the lndclient version bump?

Reply:
+1, fully understand the diff but missing rationale/context

Reply:
Added some rationale to the commit message.